src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -470,6 +470,11 @@ protected virtual void Initialize()
                 var functionMetadata = ReadFunctionMetadata(ScriptConfig, TraceWriter, _startupLogger, FunctionErrors, _settingsManager);
                 var usedBindingTypes = DiscoverBindingTypes(functionMetadata);
 
+                if (!ScriptConfig.LazyLoadExtensions)
```

Comment:
Can't we skip the above DiscoverBindingTypes call in this case? Seems like you're doing work that you're just throwing away here.